content/docs/howto/nodejs.md

```diff
@@ -271,60 +271,13 @@ BP_LAUNCHPOINT="./src/launchpoint.js"
-buildpacks will build source code into production-ready static assets, then
-automatically configure a web server to serve those assets. Check the Paketo samples
-repository for an example [React app][sample/react-servers] with build instructions.
-
-1. Define a script under the `"scripts"` property of your `package.json` that
-   builds your production-ready static assets. Most frameworks bootstrap this
-   automatically. For React, it's `"build"`.
-
-2. Find out where static assets are stored after the build script runs. It'll
-   be a directory under the root of the app directory. For React, this is
-   `./build` by default.
-
-3. Select which web server to use: NGINX or HTTPD.
-
-4. **[NGINX]** If you chose HTTPD, **skip to step 5**. Use the Node.js and
-   NGINX buildpacks to build your app. Use environment variables to configure
-   the server. `BP_NODE_RUN_SCRIPTS` should be set to the name of the _build
-   script_ from step 1. `BP_WEB_SERVER_ROOT` should be set to the build _output
-   directory_ from step 2. To (optionally) further adjust the behaviour of the
-   NGINX server, see the NGINX How-to [guides][nginx/configure].
-<!-- spellchecker-disable -->
-{{< code/copyable >}}
-pack build frontend-nginx --buildpack paketo-buildpacks/nodejs \
-                          --buildpack paketo-buildpacks/nginx \
-                          --env BP_NODE_RUN_SCRIPTS=build \
-                          --env BP_WEB_SERVER=nginx \
-                          --env BP_WEB_SERVER_ROOT=build
-{{< /code/copyable >}}
-<!-- spellchecker-enable -->
-
-5. **[HTTPD]** If you chose NGINX, **skip to step 6**. Use the Node.js and
-   HTTPD buildpacks to build your app. Use environment variables to configure
-   the server. `BP_NODE_RUN_SCRIPTS` should be set to the name of the _build
-   script_ from step 1. `BP_WEB_SERVER_ROOT` should be set to the build _output
-   directory_ from step 2. To (optionally) further adjust the behaviour of the
-   HTTPD server, see the HTTPD How-to [guides][httpd/configure].
-<!-- spellchecker-disable -->
-{{< code/copyable >}}
-pack build frontend-httpd --buildpack paketo-buildpacks/nodejs \
-                          --buildpack paketo-buildpacks/httpd \
-                          --env BP_NODE_RUN_SCRIPTS=build \
-                          --env BP_WEB_SERVER=httpd \
-                          --env BP_WEB_SERVER_ROOT=build
-{{< /code/copyable >}}
-<!-- spellchecker-enable -->
-
-6. You're done! The resulting app container will serve your frontend app with
-   an automatically generated server configuration.
+## Build and Serve a Frontend Framework App
 
+If you are using a framework that generates a static site for your (e.g. React,
+Vue, Angular) and you would like to have the buildpacks to run the build
+process and then serve using a static web server. The [Paketo Web Servers
+buildpack][web-servers] has a [frontend application
+workflow][web-servers/frontend] that covers this use case.
```

Comment:
```suggestion
If you are using a framework that generates a static site from JavaScript source
code (e.g. React, Vue, Angular), you can use the [Paketo Web Servers
buildpack][web-servers] to build the static assets and automatically configure
a web server. The web servers [documentation][web-servers/frontend] covers this use case.
```

Reply:
THANK YOU. I could not for the life of me think of good formatting for this!